index.html

```diff
@@ -55,11 +55,21 @@
           // https://www.nve.no/reguleringsmyndigheten/kunde/stroem/spoersmaal-og-svar-om-stroemstoette-for-hoeye-stroempriser/stroemstoette-for-husholdninger/
           const prisInkStøtte = prisInkMva - (s.pris > 70 ? (s.pris - 70) * 0.90 : 0) * 1.25;
 
+          // calculates "energiledd" using prices and times from
+          // https://www.bkk.no/alt-om-nettleie/nettleiepriser-for-privatkunder
+          const isDay = date.getHours() >= 6 && date.getHours() < 22;
```

Comment:
This will be off if viewing from a browser with a different TZ, such as a cell phone when travelling.

An ugly work-around that might work is some string parsing of:
```
(new Date()).toLocaleString(undefined, { timeZone: "Europe/Oslo" })
```

to determine if we're within daytime/nighttime pricing hours. Strange that browsers have a `toLocaleString` that supports tzdb strings, but no useful methods on `Date` :/

Not that this is blocking, as it's still better than what we have now, and the target audience is mostly using `Europe/Oslo` :)

Reply:
Or wait, I suppose this date comes from the json data, so I'm wrong. Ignore me please :)

Reply:
It's put into `new Date`, so I think the local timezone will mess things up (if not Oslo)

Reply:
Checked, it ignores the local tz completely, since the input string lacks tz.